vaultpress.php

```diff
@@ -3014,7 +3014,7 @@ function get_key_via_jetpack( $already_purchased = false ) {
 		}
 
 		// For version of Jetpack prior to 7.7.
-		if ( ! class_exists( 'Jetpack_IXR_Client' ) ) {
+		if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' )  ) {
```

Comment:
Would it make sense to keep the previous class_exists conditional around as well?

```suggestion
		if ( defined( 'JETPACK__VERSION' ) && version_compare( JETPACK__VERSION, '7.7', '<' ) && ! class_exists( 'Jetpack_IXR_Client' ) ) {
```